.github/CODEOWNERS

```
@@ -1,20 +1,19 @@
-# general codeowners for all files
-# (Order matters. This needs to be at the top)
-* @nvidia/cccl-codeowners
-
 # Libraries
-thrust/ @nvidia/cccl-thrust-codeowners @nvidia/cccl-codeowners
-cub/ @nvidia/cccl-cub-codeowners @nvidia/cccl-codeowners
-libcudacxx/ @nvidia/cccl-libcudacxx-codeowners @nvidia/cccl-codeowners
+thrust/ @nvidia/cccl-thrust-codeowners
+cub/ @nvidia/cccl-cub-codeowners
+libcudacxx/ @nvidia/cccl-libcudacxx-codeowners
 cudax/ @nvidia/cccl-cudax-codeowners
 c/ @nvidia/cccl-c-codeowners
```

I think Python team should be a stakeholder here 
```suggestion
c/ @nvidia/cccl-c-codeowners @nvidia/cccl-python-codeowners
```

---

(alternatively, we could just nuke `cccl-c-codeowners`...)

---

I agree there's a lot of overlap, but I've just encoded that in the membership of the two different groups. Putting two teams listed as codeowners would end up requiring 4 total reviews which is probably too many.

---

I thought the threshold is set independently in the repo setting, not based on the number of teams? (I could be wrong.)